debezium-core/src/main/java/io/debezium/util/NumberConstants.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.util;
+
+
+import java.math.BigDecimal;
+import java.nio.ByteBuffer;
+import java.util.Date;
+
+/**
+ * A set of Numbers.
+ *
+ * @author pmx
+ */
+public class NumberConstants {
+
+    public static final Short SHORT_TRUE = new Short((short) 1);
+    public static final Short SHORT_FALSE = new Short((short) 0);
+
+    public static final Integer INTEGER_TRUE = new Integer(1);
+    public static final Integer INTEGER_FALSE = new Integer(0);
+
+    public static final Long LONG_TRUE = new Long(1L);
+    public static final Long LONG_FALSE = new Long(0L);
+
+    public static final Float FLOAT_TRUE = new Float(1.0);
+    public static final Float FLOAT_FALSE = new Float(0.0);
+
+    public static final Double DOUBLE_TRUE = new Double(1.0d);
+    public static final Double DOUBLE_FALSE = new Double(0.0d);
+
+    public static final byte[] BYTE_ZERO = new byte[0];
+
+    public static final Date DATE_EPOCH_ZERO = new Date(0L);
+
+    public static final BigDecimal BIGDECIMAL_ZERO = BigDecimal.valueOf(0);
+    public static final BigDecimal BIGDECIMAL_ONE = BigDecimal.valueOf(1);
+
+    public static final ByteBuffer BYTEBUFFER_ZERO = ByteBuffer.wrap(new byte[0]);
+
+    /**
+     * Convert boolean object to bigDecimal object.
+     *
+     * @param data a boolean object
+     * @return bigDecimal 0 or 1
+     * @throws NullPointerException If {@code data} is {@code null}
+     */
+    public static BigDecimal getBigDecimal(Boolean data) {
```

Comment:
Given we have those methods here, I think the class shouldn't be named `...Constants` (I'd keep that for interfaces solely containing constants). How about something like `NumberConversions`?